src/views/virtualmachines/details/tabs/configuration/ssh/SSHTab.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC, useCallback } from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+
+import VirtualMachineModel from '@kubevirt-ui/kubevirt-api/console/models/VirtualMachineModel';
+import { V1VirtualMachine, V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import SidebarEditor from '@kubevirt-utils/components/SidebarEditor/SidebarEditor';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { PATHS_TO_HIGHLIGHT } from '@kubevirt-utils/resources/vm/utils/constants';
+import { k8sUpdate } from '@openshift-console/dynamic-plugin-sdk';
+import { Grid, GridItem, PageSection, Stack, Title } from '@patternfly/react-core';
+
+import SSHTabAuthorizedSSHKey from './components/SSHTabAuthorizedSSHKey';
+import SSHTabSSHAccess from './components/SSHTabSSHAccess';
+
+type SSHTabProps = RouteComponentProps<{
+  name: string;
+  ns: string;
+}> & {
+  obj?: V1VirtualMachine;
+  vmi?: V1VirtualMachineInstance;
+};
+
+const SSHTab: FC<SSHTabProps> = ({ obj: vm, vmi }) => {
+  const { t } = useKubevirtTranslation();
+
+  const onSubmit = useCallback(
+    (updatedVM: V1VirtualMachine) =>
+      k8sUpdate({
+        data: updatedVM,
+        model: VirtualMachineModel,
+        name: updatedVM?.metadata?.name,
```

Comment:
Selector candidate here and the next line

Reply:
ah?